projects/js-packages/components/components/contextual-upgrade-trigger/index.tsx

```diff
@@ -10,10 +10,12 @@ const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( {
 	cta,
 	onClick,
 	href,
+	openInNewTab = false,
 	className,
 } ) => {
 	const Tag = href !== undefined ? 'a' : 'button';
-	const tagProps = Tag === 'a' ? { href, target: '_blank' } : { onClick };
+	const tagProps =
+		Tag === 'a' ? { href, ...( openInNewTab ? { target: '_blank' } : {} ) } : { onClick };
```

Comment:
Super minor, but I think this could be:
```suggestion
		Tag === 'a' ? { href, ...( openInNewTab && { target: '_blank' } ) } : { onClick };
```

I'm not sure if it simplifies it or not!

Reply:
Just tested to be sure and it works well, sold!